src/TypeScript/Rules/RuleDefinitionsProvider.cs

```diff
@@ -20,33 +20,22 @@
 
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
+using System.Diagnostics;
 using System.IO;
 using System.Linq;
 using System.Text;
 using Newtonsoft.Json;
 
 namespace SonarLint.VisualStudio.TypeScript.Rules
 {
```

Comment:
This class now implements four interfaces. I put the interfaces in separate files to reduce the noise here.